weak-modules

```diff
@@ -95,7 +95,7 @@ read_modules_list() {
 	    module_krels[n]=$(krel_of_module "${modules[n]}")
 	else
 	    # Try to extract the kernel release from the path
-	    set -- "${modules[n]#/lib/modules/}"
+	    set -- "${modules[n]#*/lib/modules/}"
```

Comment:
Simply using the `*` wildcard doesn't cut it. If we really must change our code here, we have to check whether the modules for the given kernel reside in `/usr/lib`  or in `/lib` and act accordingly. But honestly I'd prefer to stick with `/lib`.

Reply:
$ modules=(/usr/lib/modules/foo/bar.ko /usr/lib/modules/foo/baz.ko); n=1; echo ${modules[n]#*/lib/modules/}
    foo/baz.ko